Lib/xmlrpc/client.py

```diff
@@ -1421,15 +1421,15 @@ def __init__(self, uri, transport=None, encoding=None, verbose=False,
         # establish a "logical" server connection
 
         # get the url
-        type, uri = urllib.parse._splittype(uri)
-        if type not in ("http", "https"):
+        p = urllib.parse.urlparse(uri)
+        if p.scheme not in ("http", "https"):
             raise OSError("unsupported XML-RPC protocol")
-        self.__host, self.__handler = urllib.parse._splithost(uri)
+        self.__host, self.__handler = p.netloc, p.path
```

Comment:
No need to write it in one line now.
```suggestion
        self.__host = p.netloc
        self.__handler = p.path or "/RPC2"
```

Reply:
@serhiy-storchaka 
Thanks for the review
It looks more clear and readable.